src/main/java/prefs/Preferences.java

```diff
@@ -52,15 +52,17 @@ public static Preferences create(String configFileName) {
     }
 
     public void saveGlobalConfig() {
-        fileHandler.saveGlobalConfig(global);
+        globalConfigFile.saveConfig(global);
     }
 
     private void initGlobalConfig() {
-        global = fileHandler.initGlobalConfig();
+        GlobalConfig emptyGlobalConfig = new GlobalConfig();
+        globalConfigFile.saveConfig(emptyGlobalConfig);
+        loadGlobalConfig();
```

Comment:
I am not sure about this method. It seems to be roundabout that making an empty global config needs to be done via a newly created file. Wouldn't instancing global here as new GlobalConfig() then save it suffices? Unless I am missing something.